File: Suhyun/Level1/2월 1주차/행렬의 덧셈/Plus_array.java
import java.util.Arrays;
public class Plus_array {
    public static void main(String[] args) {
    
    int [][] arr1={{3},{4}};
    int [][] arr2={{1},{2}};

    int[][] answer = new int[arr1.length][arr2[0].length];

    for(int i=0; i<arr1.length ;i++){
        for(int j=0; j<arr2[0].length; j++){

            answer[i][j]=arr1[i][j]+arr2[i][j];
        }
    }
    System.out.println(Arrays.toString(answer));
    
    }
}

File: Suhyun/Level1/2월 1주차/행렬의 덧셈/Plus_array.js
function solution(arr1, arr2) {
    let answer=new Array(arr1.length);
    
    for(let i=0; i<arr1.length; i++ ){
        answer[i]=new Array(0);
        for(let j=0; j<arr2[0].length; j++){
            answer[i].push(arr1[i][j]+arr2[i][j]);
        }
    }
    
    return answer;
}